gui/src/components/HomeComponents/create-project.module.css

```diff
@@ -0,0 +1,5 @@
+.integrate_github_container {
+    background-color: #262628;
```

Comment:
#262628 is being used multiple places, so as previously said please define it in app.css and use it from there